src/services/syncFeedPerUser.js

```diff
@@ -236,7 +236,7 @@ const syncFeedPerUser = async (req, res) => {
 module.exports = {
   syncFeedPerUser,
   syncFeedPerUserProcess,
-  syncMainFeedBroadPerUser,
+  // syncMainFeedBroadPerUser,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The export of `syncMainFeedBroadPerUser` has been commented out. If this function is no longer needed, it's better to remove this line completely.

```diff
- // syncMainFeedBroadPerUser,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->